spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+describe "POST api/v1/targets", type: :request do
+
+  let(:topic_list) { create_list(:topic, 5) } 
+  let(:user) { create(:user) }
+
+  let(:params) do {
+    user: {
+      email: user.email,
+      password: user.password
+    }    
+  }
+  end
+
+  before do
+    user.confirm
+    post new_user_session_path, params: params, as: :json
```

Comment:
This is going to be used many times in the request specs.
Maybe we can find out another way for doing this?

What about using a spec helper, that defines `authenticated_headers`. And sets the correct headers for a log in user.

So in specific request specs (like this file) we only do something like 
`post api_v1_targets_path, params: target_params, headers: authenticated_headers, as: :json `

Makes sense?